src/components/Modal/CartModal/CartModal.tsx

```diff
@@ -1,39 +1,97 @@
 
-import { DELIVERY_PRICE, MODALS } from '../../../constants'
-import { CART_FORM_COMPONENTS, ORDER_TYPE, ORDER_TYPE_BUTTONS, VALIDATIONS } from '../../../constants/forms'
+import { DELIVERY_PRICE, MODALS, VALID_LOCALS } from '../../../constants'
+import {
+  CART_FORM_COMPONENTS,
+  ORDER_FROM,
+  ORDER_FROM_FORMS,
+  ORDER_FROM_INPUTS,
+  ORDER_TYPE,
+  VALIDATIONS,
+} from '../../../constants/forms'
 import { CartContext } from '../../../store/Cart/Cart.context'
 import { ModalContext } from '../../../store/Modal/Modal.context'
-import { OrderType } from '../../../types'
+import { OrderPayment, OrderType } from '../../../types'
+import { sendProductsToCMS } from '../../../utils/products'
 import Button from '../../Button/Button'
 import Input from '../../Input/Input'
-import RadioButton from '../../RadioButton/RadioButton'
 import SelectedProduct from '../../SelectedProduct/SelectedProduct'
 import Bag from '../../Svgs/Bag/Bag'
 import styles from './CartModal.module.scss'
 import FormBlock from './FormBlock/FormBlock'
+import { CartState } from '../../../types/state'
+
+const sendMailOrder = async (
+  data: Record<string, string | boolean>,
+  products: CartState['groupedByToppingsProducts'],
+  price: number,
+  submitLocal: string,
+  isValidLocal: boolean
+) => {
+  try {
+    await fetch('/api/order', {
+      method: 'POST',
+      headers: { 'content-type': 'application/json' },
+      body: JSON.stringify({
+        data: {...data, price},
+        products,
+        submitLocal,
+        isValidLocal,
+      }),
+    })
+  } catch(error){
+    return 0
+  }
+}
 
 const CartModal = () => {
-  const { register, handleSubmit, formState: { errors } } = useForm()
+
+  const { register, handleSubmit, setValue, formState: { errors } } = useForm()
 
   const { state: { show }, actions: { closeModal } } = useContext(ModalContext)
   const { state: { price, groupedByToppingsProducts } } = useContext(CartContext)
 
-  const [orderType, setOrderType] = useState<OrderType>('')
+  const local = window.location.host.split('.')[0]
+  const isValidLocal = local === VALID_LOCALS.botanica || local === VALID_LOCALS.rascanovca
+  const inputValidations = isValidLocal ? VALIDATIONS.masa : VALIDATIONS.email
+
+  const [orderType, setOrderType] = useState<OrderType>('takeaway')
+  const [orderPayment, setOrderPayment] = useState<OrderPayment>('')
+
+  const { t } = useTranslation('cart')
 
   const onSubmit = (data: Record<string, string | boolean>) => {
-    // eslint-disable-next-line no-console
-    console.table(data)
+
+    const getTakeawayLocation = (location: string) => {
+      switch (location){
+        case 'Telecentru' :
+          return 'manager.mister.'
+        case 'Malina Mică' :
+          return 'manager.mister.'
+        case 'Râșcani' :
+          return 'rascanovca.mr.'
+        case 'Botanica' :
+          return 'botanica.mr.'
+        default :
+          return 'manager.mister.'
+      }
```

Comment:
I think `lookup object` > `switch statement` for simple cases.
I would add in const:
```
const TAKEAWAY_LOCATIONS = {
    'Telecentru': 'manager.mister.',
    'Malina Mică': 'manager.mister.',
    ...
}
```
and use it in code like this: `TAKEAWAY_LOCATIONS[location] `